src/full/Agda/TypeChecking/Coverage/Cubical.hs

```diff
@@ -86,7 +86,7 @@ createMissingIndexedClauses f n x old_sc scs cs = do
          reportSDoc "tc.cover.indexed" 20 $ text "size (xs,infos):" <+> pretty (size xs,size infos)
          reportSDoc "tc.cover.indexed" 20 $ text "xs :" <+> pretty (map fst xs)
 
-         unless (size xs == size infos + 1) $
+         unless (natSize xs == 1 + natSize infos) $
```

Comment:
Is this test expected to succeed?

Reply:
I suppose so, but don't know for sure.  And it is only for debug printing, so I guess I can revert this change.
Good catch!

Reply:
Reverted this change.